mlflow/protos/mlflow_artifacts.proto

```diff
@@ -0,0 +1,86 @@
+syntax = "proto2";
+
+package mlflow.artifacts;
+
+import "scalapb/scalapb.proto";
+import "databricks.proto";
+
+option java_package = "org.mlflow.api.proto";
+option py_generic_services = true;
+option (scalapb.options) = {
+  flat_package: true,
+};
+
+service MlflowArtifactsService {
+
+  rpc downloadArtifact (DownloadArtifact) returns (DownloadArtifact.Response) {
+    option (rpc) = {
+      endpoints: [{
+        method: "GET",
+        path: "/mlflow-artifacts/artifacts/<path:artifact_path>",
+        since { major: 2, minor: 0 },
+      }],
+      visibility: PUBLIC,
+      rpc_doc_title: "Download Artifact",
+    };
+  }
+
+  rpc uploadArtifact (UploadArtifact) returns (UploadArtifact.Response) {
+    option (rpc) = {
+      endpoints: [{
+        method: "PUT",
+        path: "/mlflow-artifacts/artifacts/<path:artifact_path>",
+        since { major: 2, minor: 0 },
+      }],
+      visibility: PUBLIC,
+      rpc_doc_title: "Upload Artifact",
+    };
+  }
+
+  rpc listArtifacts (ListArtifacts) returns (ListArtifacts.Response) {
+    option (rpc) = {
+      endpoints: [{
+        method: "GET",
+        path: "/mlflow-artifacts/artifacts",
+        since { major: 2, minor: 0 },
+      }],
+      visibility: PUBLIC,
+      rpc_doc_title: "List Artifacts",
+    };
+  }
+}
+
+message DownloadArtifact {
+  option (scalapb.message).extends = "com.databricks.rpc.RPC[$this.Response]";
+
+  message Response {}
+}
+
+message UploadArtifact {
+  option (scalapb.message).extends = "com.databricks.rpc.RPC[$this.Response]";
+
+  message Response {}
+}
+
+message ListArtifacts {
+  option (scalapb.message).extends = "com.databricks.rpc.RPC[$this.Response]";
```

Comment:
If we don't intend to implement these on Databricks, then I think we can remove the `(scalapb.message).extends` option from each of these protos.

Reply:
Makes senes. Let's remove them.